pkg/api/types.go

```diff
@@ -334,6 +467,8 @@ type Capabilities struct {
 type ResourceRequirements struct {
 	// Limits describes the maximum amount of compute resources required.
 	Limits ResourceList `json:"limits,omitempty"`
+	// Requests describes the minimum amount of compute resources required.
+	Requests ResourceList `json:"requests,omitempty"`
```

Comment:
Brian and I spoke.  I still find the semantic weird, but it is what we have documented, so let's run with it.  I withdraw my objection here.